testsuite/tests/warnings/w32.ml

```diff
@@ -70,6 +70,23 @@ module type S = sig
   module F:  sig val x : int end -> sig end
 end
 
+(* from ocaml/ocaml#13955 no unused warning should be triggered for [test] *)
+
+module I : sig
```

Comment:
I propose to disable warning 60 for this group of module to reduce the noise in the test.